bazel/README.md

```diff
@@ -18,6 +18,9 @@ emsdk_deps()
 load("@emsdk//:emscripten_deps.bzl", emsdk_emscripten_deps = "emscripten_deps")
 emsdk_emscripten_deps(emscripten_version = "2.0.31")
 
+load("@emsdk//:emscripten_cache.bzl", emsdk_emscripten_cache = "emscripten_cache")
+emsdk_emscripten_cache()
+
```

Comment:
Do all users need to specify this?  I would hope that users who want to default cache can skip this?   In which case we can delete these two lines?

Reply:
I had a quick look and I suspect that it would require a new approach to solving the problem (and I don't really have the time for that right now).

Considering that [Bzlmod is replacing the WORKSPACE system](https://bazel.build/external/migration), I would like to ask to have this merged as is and when it comes time to transition emsdk to Bzlmod, we can find a new way to initialize emsdk and all of its components from the MODULE file, which I believe is more flexible.

Reply:
That seems unfortunate since won't all existing users now need to add these two lines?

Reply:
I slept on it and came up with another solution which works around this problem. See the latest commit, but in essence, we can move the cache initialization inside of the toolchain registration and use a default argument to maintain backwards compatibility. Let me know what you think!